plugin/db/util/driverutil.go

```diff
@@ -306,18 +306,29 @@ func Query(ctx context.Context, l *zap.Logger, db *sql.DB, statement string, lim
 	}
 
 	colCount := len(columnTypes)
+
+	columnTypeNames := make([]string, colCount)
+	for i, v := range columnTypes {
+		// DatabaseTypeName returns the database system name of the column type.
+		// refer: https://pkg.go.dev/database/sql#ColumnType.DatabaseTypeName
+		columnTypeNames[i] = strings.ToUpper(v.DatabaseTypeName())
+	}
+
 	rowCount := 0
 	data := []interface{}{}
 	for rows.Next() {
 		scanArgs := make([]interface{}, colCount)
-		for i, v := range columnTypes {
-			switch v.DatabaseTypeName() {
+		for i, v := range columnTypeNames {
+			// TODO(steven need help): Consult a common list of data types from database driver documentation. e.g. MySQL,PostgreSQL.
+			switch v {
 			case "VARCHAR", "TEXT", "UUID", "TIMESTAMP":
 				scanArgs[i] = new(sql.NullString)
 			case "BOOL":
 				scanArgs[i] = new(sql.NullBool)
-			case "INT4":
+			case "INT", "INTEGER":
```

Comment:
Is this consistent across databases, at least MySQL and Postgres?

But this change LGTM.

Reply:
I can't find the "INT4" type from the databases we use, at least MySQL and PostgreSQL. Maybe it's a typo. @tianzhou

Reply:
INT, INTEGER is better